src/e2e/java/teammates/e2e/cases/lnp/InstructorUpdateLNPTest.java

```diff
@@ -0,0 +1,367 @@
+                                .withQuestionDescription(FEEDBACK_QUESTION_TEXT)
+                                .withCourseId(COURSE_ID)
+                                .withQuestionDetails(details)
+                                .withQuestionNumber(1)
+                                .withGiverType(FeedbackParticipantType.INSTRUCTORS)
+                                .withRecipientType(FeedbackParticipantType.STUDENTS)
+                                .withShowResponsesTo(showResponses)
+                                .withShowGiverNameTo(showGiverName)
+                                .withShowRecipientNameTo(showRecepientName)
+                                .build()
+                );
+
+                return feedbackQuestions;
+            }
+
+            @Override
+            protected Map<String, FeedbackResponseAttributes> generateFeedbackResponses() {
+                Map<String, FeedbackResponseAttributes> feedbackResponses = new HashMap<>();
+                FeedbackTextResponseDetails details =
+                        new FeedbackTextResponseDetails(FEEDBACK_RESPONSE_ID);
+
+                feedbackResponses.put(FEEDBACK_RESPONSE_ID,
+                        FeedbackResponseAttributes.builder(FEEDBACK_QUESTION_ID,
+                            INSTRUCTOR_EMAIL,
+                            STUDENT_EMAIL)
+                            .withCourseId(COURSE_ID)
+                            .withFeedbackSessionName(FEEDBACK_SESSION_NAME)
+                            .withGiverSection(GIVER_SECTION_NAME)
+                            .withRecipientSection(RECEIVER_SECTION_NAME)
+                            .withResponseDetails(details)
+                            .build());
+
+                return feedbackResponses;
+            }
+
+            @Override
+            protected Map<String, FeedbackResponseCommentAttributes> generateFeedbackResponseComments() {
+                ArrayList<FeedbackParticipantType> showComments = new ArrayList<>();
+                showComments.add(FeedbackParticipantType.RECEIVER);
+                showComments.add(FeedbackParticipantType.INSTRUCTORS);
+
+                ArrayList<FeedbackParticipantType> showGiverName = new ArrayList<>();
+                showGiverName.add(FeedbackParticipantType.INSTRUCTORS);
+
+                Map<String, FeedbackResponseCommentAttributes> feedbackResponseComments = new HashMap<>();
+
+                for (int i = 1; i <= NUMBER_OF_FEEDBACK_RESPONSE_COMMENTS; i++) {
+                    String responseCommentText = FEEDBACK_RESPONSE_COMMENT_ID + i;
+                    feedbackResponseComments.put(responseCommentText,
+                            FeedbackResponseCommentAttributes.builder()
+                                .withCourseId(COURSE_ID)
+                                .withFeedbackResponseId(FEEDBACK_RESPONSE_ID)
+                                .withFeedbackQuestionId(FEEDBACK_QUESTION_ID)
```

Comment:
Similarly here.
For feedback response ID, the convention is `(feedback_question_id)%(giver_email)%(recipient_email)`, but I don't think this one matters for now.